python-wrapper/tests/conftest.py

```diff
@@ -43,7 +43,9 @@ def aura_ds_instance() -> Generator[Any, None, None]:
 
     # setting as environment variables to run notebooks with this connection
     os.environ["NEO4J_URI"] = dbms_connection_info.uri
+    assert isinstance(dbms_connection_info.username, str)
```

Comment:
right latest gds release changed this.
There is `dbms_connection_info.get_auth()`, but thats not useful here to set an env variable